spacy/ml/extract_spans.py

```diff
@@ -32,8 +32,8 @@ def forward(
         Y = Ragged(X.dataXd[indices], spans.dataXd[:, 1] - spans.dataXd[:, 0])  # type: ignore[arg-type, index]
     else:
         Y = Ragged(
-            ops.xp.zeros(X.dataXd.shape, dtype=X.dataXd.dtype),
-            ops.xp.zeros((len(X.lengths),), dtype="i"),
+            ops.xp.zeros((0, 0), dtype=X.dataXd.dtype),
+            ops.xp.zeros((0,), dtype="i"),
```

Comment:
This is now never reached in `spancat`. The previous placeholder was clearly incorrect. This shouldn't crash the model now, but it's also not really necessary. You could just require that `len(indices) > 0` instead.

Reply:
Nevermind about this, this was wrong.